### ISSUE TITLE
Don't show settings window at logon

### ISSUE THREAD

**Issue by eugenesvk** [tekezo/ShowyEdge#7]
Is it possible to autohide settings window at OSX logon?
I don't need to see it every time I restart my Mac, I've already set it all up :)
Thank you!

OSX 10.11, ShowyEdge 3.1.3

**Comment by tekezo** [comment]
I cannot reproduce this issue.
Could you show me your login items in System Preferences > Users & Groups?
![160428-0001](https://cloud.githubusercontent.com/assets/659178/14874343/123be994-0d3a-11e6-968e-4db148d94304.png)

**Comment by eugenesvk** [comment]
Thanks for the screenshot. I've removed your app from Startupizer (launcher manager that is able to delay launching apps) and moved it back to the `Users & Groups` login items with **Hide** field **unckecked** and now it's all fine.
It's a bit counterintuitive though, because when  I tried to check the Hide field, I'd instead actually see a popup ShowyEdge window settings on start, but whatever.
Thanks
